node_modules/.package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This `node_modules` seems to be outside of the frontend folder, It can be removed  :)